.crossbar/config.yaml

```diff
@@ -35,6 +35,7 @@ workers:
     endpoint:
       type: tcp
       port: 20408
+      user_timeout: 5000
```

Comment:
Is this a behavior change triggered by the new crossbar?
Note that TCP_USER_TIMEOUT also applies during the normal connection lifetime, not just shutdown.

We already ping the exporters regularly and kick them from the crossbar on timeout. Is this broken with the new crossbar release?

Reply:
Context is https://github.com/crossbario/crossbar/pull/1930.

> Is this a behavior change triggered by the new crossbar?

I don't think so.

> Note that TCP_USER_TIMEOUT also applies during the normal connection lifetime, not just shutdown.
> 
> We already ping the exporters regularly and kick them from the crossbar on timeout. Is this broken with the new crossbar release?

This is especially useful for [LXA Test Automation Controller](https://www.linux-automation.com/en/products/lxa-tac.html), which might not get shut down cleanly. The TCP connection is considered alive for quite a long time.

I thought that this might be a sane default. Do you have concerns?

Reply:
This seems unrelated to the pyproject.toml migration.

I'm still not clear on why we'd need both the TCP timeout and the ping/exporter-kick at the same time.

Reply:
I dropped this because we decided to drop the version bumps and dependency relaxations in this particular PR altogether.